tests/test_oc4ids_tranforms.py

```diff
@@ -0,0 +1,1332 @@
+      "id": "1",
+      "contractingProcesses": [
+        {
+          "id": "ocds-213czf-1",
+          "summary": {
+            "ocid": "ocds-213czf-1"
+          },
+          "releases": [
+            {
+              "url": "example.com#1",
+              "date": "2001-02-03T04:05:06Z",
+              "tag": "planning"
+            }
+          ]
+        },
+        {
+          "id": "ocds-213czf-2",
+          "summary": {
+            "ocid": "ocds-213czf-2"
+          },
+          "releases": [
+            {
+              "url": "example.com#2",
+              "date": "2001-02-03T04:05:06Z",
+              "tag": "planning"
+            }
+          ]
+        }
+      ]
+    }
+    """
+
+    assert output == json.loads(expected)
+
+
+def test_procuring_entity():
+
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "tender": {"procuringEntity": {"id": 1}},
+            "parties": [{"id": "1", "roles": ["procuringEntity"]}],
+        },
+    ]
+
+    output = transforms._run_transforms(
+        copy.deepcopy(releases), "1", transforms=[transforms.contracting_process_setup, transforms.procuring_entity],
+    )
+
+    assert output["parties"] == releases[0]["parties"]
+    assert output["contractingProcesses"][0]["summary"]["tender"] == releases[0]["tender"]
+
+
+def test_administrative_entity():
+
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "parties": [{"id": "1", "name": "a", "roles": ["administrativeEntity"]}],
+        },
+    ]
+
+    output = transforms._run_transforms(
+        copy.deepcopy(releases),
+        "1",
+        transforms=[transforms.contracting_process_setup, transforms.administrative_entity],
+    )
+
+    assert output["parties"] == releases[0]["parties"]
+    assert output["contractingProcesses"][0]["summary"]["tender"]["administrativeEntity"] == {"id": "1", "name": "a"}
+
+
+def test_multiple_administrative_entity_in_process():
+    releases = [
+        {
+            "ocid": "ocds-213czf-1",
+            "id": "1",
+            "tag": "planning",
+            "date": "2001-02-03T04:05:06Z",
+            "parties": [
+                {"id": "1", "name": "a", "roles": ["administrativeEntity"]},
+                {"id": "2", "name": "b", "roles": ["administrativeEntity"]},
+            ],
+        },
+    ]
+
+    output = transforms._run_transforms(
+        copy.deepcopy(releases),
+        "1",
+        transforms=[transforms.contracting_process_setup, transforms.administrative_entity],
+    )
+
+    assert output["parties"] == releases[0]["parties"]
+
+    # tender is not created as there are multiple adminastrative entities
+    assert "tender" not in output["contractingProcesses"][0]["summary"]
+
+
+def test_contract_status_pre_award():
```

Comment:
Should this test include some contracting processes which evaluate to 'active' or 'closed'?